playbooks/deploy_ntp.yml

```diff
@@ -1,8 +1,12 @@
 - name: Setup NTP
   hosts: ntp_host
-  gather_facts: "{{ (setup_ntp_service | default(True)) | bool }}"
+  gather_facts: False
   vars:
     SETUP_NTP_SERVICE: "{{setup_ntp_service | default(True)}}"
   roles:
     - role: setup_ntp
       when: SETUP_NTP_SERVICE == True
+  pre_tasks:
```

Comment:
Can you put the pre_tasks before the roles.

Reply:
Oh snap! I can't believe I missed this one here. @nocturnalastro good catch.

Reply:
There you go! Thx for the feedback!